.lando.yml

```diff
@@ -24,8 +24,8 @@ services:
       - composer install
       - mkdir -p ~/.drush/site-aliases
       - ln -sf /app/drush/nnels-local.aliases.drushrc.php ~/.drush/site-aliases/nnels-local.drushrc.php
-      - drush dl composer_generate
-    run_as_root: updatedb
+      # - drush dl composer_generate
```

Comment:
This was the first error that I came across:

```
No release history available for composer_generate 8.x.                                                                                                                                                                                              
                                                         [error]
Could not download requested project(s).                                                                                                                                                                                                                                                                                                                               
                                                         [error]
ERROR ==> Could not download requested project(s).                                                                                                                                                                                                                                                                                
                                                                   [error]
```

For some reason composer is trying to download the 8.x version and it's only available for D7. I didn't dig into why that was happening since I don't think it's required - we already have a composer.json/composer.lock file.

Reply:
I used composer_generate initially to create the json + lock. It's probably not needed now as you say.